### PR TITLE
docs(transform): enhance the transform/pauseResume documents.

### PR DESCRIPTION
**Pull request by jongmoon**
## Issue
#299 

## Details
enhance the transform/pauseResume documents.

## Preferred reviewers
@naver/egjs-dev

### CODE REVIEW

File: src/hook/pauseResume.js [hunk 1]
```diff
@@ -182,7 +182,7 @@ eg.module("pauseResume", ["jQuery"], function($) {
 
 	/**
 	 * Pause animation
-	 * @ko 에니메이션을 일시 정지한다
+	 * @ko <a href="http://api.jquery.com/animate/">jQuery animate 메소드</a>에 의해 수행되고 있는 에니메이션을 일시 정지한다.
```
Comment (mixed) [inline src/hook/pauseResume.js]: suggest
`jQuery animate 메소드가 실행한 에니메이션을 일시 정지한다.`

File: src/hook/pauseResume.js [hunk 2]
```diff
@@ -214,7 +214,7 @@ eg.module("pauseResume", ["jQuery"], function($) {
 
 	/**
 	 * Resume animation
-	 * @ko 애니메이션을 재개한다
+	 * @ko <a href="http://naver.github.io/egjs/latest/doc/jQuery.html#pause">pause 메소드</a>에 의해 일시 정지된 애니메이션을 재개한다.
```
Comment (mixed) [inline src/hook/pauseResume.js]: `jQuery animate 메소드가 일시 정지한`

File: src/hook/transform.js [hunk 3]
```diff
@@ -5,7 +5,7 @@
 
 /**
  * Extends jQuery animate in order to use 'transform' property
- * @ko jQuery animate 사용시 transform을 사용할 수 있도록 확장한 animate 메소드
+ * @ko <a href="http://api.jquery.com/animate/">jQuery animate 메소드</a>를 확장하여 transform 속성을 사용가능하게 한다. jQuery animate 와 완전히 동일한 인터페이스를 갖는다.
```
Comment (mixed) [inline src/hook/transform.js]: `확장하여 transform 속성을 사용가능하게 한다.`
->
`확장하여 transform 속성이 가능하다.`

`jQuery animate 와 완전히 동일한 인터페이스를 갖는다.`
->
`jQuery animate의 인터페이스와 같다.`

File: src/hook/pauseResume.js (hunk 2 above)
Comment (jongmoon) [inline src/hook/pauseResume.js]: @mixed ok, I'll apply it

File: src/hook/transform.js (hunk 3 above)
Comment (jongmoon) [inline src/hook/transform.js]: @mixed , @sculove I changed it differently as you suggest. check it please~

File: src/hook/pauseResume.js (hunk 1 above)
Comment (jongmoon) [inline src/hook/pauseResume.js]: @mixed ok, I'll apply it

**Comment by sculove** [comment]
LGTM

File: src/hook/pauseResume.js [hunk 4]
```diff
@@ -182,7 +182,7 @@ eg.module("pauseResume", ["jQuery"], function($) {
 
 	/**
 	 * Pause animation
-	 * @ko 에니메이션을 일시 정지한다
+	 * @ko <a href="http://api.jquery.com/animate/">jQuery animate 메소드</a>가 실행한 에니메이션을 일시 정지한다.
```
Comment (netil) [inline src/hook/pauseResume.js]: The expression '메소드' should be corrected as '메서드', which is recommended by tech writer team.
Check out from: http://d2.naver.com/helloworld/9323973

Comment (jongmoon) [inline src/hook/pauseResume.js]: @netil I got it~

**Comment by netil** [comment]
LGTM

**Pull request merged**